javascript/ql/experimental/adaptivethreatmodeling/src/StoredXssATM.ql

```diff
@@ -0,0 +1,29 @@
+/**
+ * For internal use only.
+ *
+ * @name Stored cross-site scripting (boosted)
+ * @description Using uncontrolled stored values in HTML allows for a stored cross-site scripting vulnerability.
+ * @kind path-problem
+ * @scored
+ * @problem.severity error
+ * @security-severity 6.1
+ * @id adaptive-threat-modeling/js/stored-xss
+ * @tags experimental experimental/atm security external/cwe/cwe-079 external/cwe/cwe-116
+ */
```

Comment:
(Also applies to the other queries)

Update these to line up with the existing queries like `XssAtm.ql`:

- Use an alert message like "(Experimental) This may be a cross-site scripting vulnerability due to $@. Identified using machine learning.", which doesn't include the score.
- Use "experimental" instead of "boosted" in the query name
- Use an `@id` like `js/ml-powered/<RHS of original query id>`
- Remove `experimental/atm` from the tags
- Rename `ATM` -> `Atm`

/cc @esbena in case this file was autogenerated, in which case we might want to file an issue to update the script that generated it.

Reply:
Thanks! I have added an issue in the scripting repo (https://github.com/github/codeql-ml-turboboost/issues/1), although that repo should be merged into https://github.com/github/ml-ql-adaptive-threat-modeling one day)

Reply:
Great, thanks!  By the way, please could you give me access to that repo?

Reply:
Great, thanks!  By the way, please could you give `github/ml-ql-adaptive-threat-modeling` access to that repo?

Reply:
Done.